### ISSUE TITLE
panic: runtime error during improper wwctl overlay chown

### ISSUE THREAD

**Issue by anderbubble** [hpcng/warewulf#484]
I got a panic when trying to use `wwctl overlay chown`. Fair enough that I used the wrong syntax for the command-line arguments; but I think it probably shouldn't panic regardless.

```
# wwctl overlay chown slurm /etc/munge munge:munge
panic: runtime error: index out of range [3] with length 3

goroutine 1 [running]:
github.com/hpcng/warewulf/internal/app/wwctl/overlay/chown.CobraRunE(0x15e8ee0, {0xc0001322d0, 0x3, 0x3})
	/builddir/build/BUILD/warewulf-4.3.0/internal/app/wwctl/overlay/chown/main.go:24 +0x57b
github.com/spf13/cobra.(*Command).execute(0x15e8ee0, {0xc0001322a0, 0x3, 0x3})
	/builddir/build/BUILD/warewulf-4.3.0/vendor/github.com/spf13/cobra/command.go:850 +0x60e
github.com/spf13/cobra.(*Command).ExecuteC(0x15e3d80)
	/builddir/build/BUILD/warewulf-4.3.0/vendor/github.com/spf13/cobra/command.go:958 +0x3ad
github.com/spf13/cobra.(*Command).Execute(...)
	/builddir/build/BUILD/warewulf-4.3.0/vendor/github.com/spf13/cobra/command.go:895
main.main()
	/builddir/build/BUILD/warewulf-4.3.0/cmd/wwctl/main.go:13 +0x25
```

**Comment by mslacken** [comment]
Could not reproduce this in 4.4.0, can you lease retry?

**Comment by anderbubble** [comment]
I've taken a note to try it again.

**Comment by anderbubble** [comment]
Agreed: appears fixed in 4.4.0.

```
$ sudo wwctl overlay chown slurm /etc/munge munge:munge
ERROR  : UID is not an integer: munge:munge
```
